signalfx-org-metrics/README.md.jinja

```diff
@@ -11,43 +11,72 @@
 
 ## DESCRIPTION
 
-Splunk Infrastructure Monitoring generates a number of metrics you can use to monitor your usage of the application, including:
+Splunk Infrastructure Monitoring provides the following types of metrics that measure your usage:
 
--  Ingest-related metrics that characterize the volume and nature of data that you are sending to Infrastructure Monitoring, such as the number of datapoints that were sent or how many metric time series were created in response to the datapoints and dimensions sent.
+-  Ingest metrics that measure the data you're sending to Infrastructure Monitoring, such as the number of datapoints you've sent, or the number of metric time series (**MTS**) created from the datapoints and dimensions you sent
 
--  Usage or engagement metrics, such as the number of dashboards or charts in the organization.
+-  App usage or engagement metrics that measure your use of application features, such as the number of dashboards or charts in your organization
 
--  Metrics that tell you how the application pulls data on your behalf from cloud services like Amazon Web Services or Google Cloud Platform, such as the number of calls to each GCP Stackdriver client method or how many of the calls to the AWS CloudWatch API are being throttled by AWS.
+-  Cloud service integration metrics that measure how Infrastructure Monitoring pulls data from cloud services such as Amazon Web Services (**AWS**) or Google Cloud Platform (**GCP**), such as the number of calls to each GCP Stackdriver client method, or the number of the calls to the AWS CloudWatch API that AWS has throttled
 
-Your organization is not charged for these metrics, and they do not count against any limits.
+-  Metrics that measure your use of resources for which you can specify limits, such as the number of custom MTS you've created. These metrics
+   help you track usage and avoid going over your organization limits.
```

Comment:
why is this split into multiple lines but other bullet points aren't?

Reply:
Lines aren't important in .md. I usually put in line breaks to help me read the source content.